tests/integration/ha_tests/helpers.py

```diff
@@ -450,11 +423,21 @@ async def reused_storage(ops_test: OpsTest, unit_name, removal_time) -> bool:
 
         item = json.loads(line)
 
+        # We need a msg
         if "msg" not in item:
             continue
 
+        # We need the attrs
```

Comment:
can you add why?

maybe something like:
`# "attr" is needed used to gather state information from mongodb`

Reply:
Sure

Reply:
mind making this change?

Reply:
Oh my bad I thought I had done it

Reply:
It has been done